paragon/_overrides.scss

```diff
@@ -68,7 +75,7 @@
     color: $body-color;
     font-weight: 400;
     &:before {
-      background-image: $icon;
+      background-image: str-replace($icon, "#", "%23");
```

Comment:
wut

Reply:
It's a weird issue. You need the `#` in the string template where `$icon` is defined in order to pipe the color into the url encoded svg, but then the `#` needs to actually be url encoded.